src/routes/api/hotels.route.js

```diff
@@ -0,0 +1,155 @@
+ *                 type: string
+ *               message:
+ *                 type: string
+ *               data:
+ *                 type: object
+ *                 properties:
+ *                   id:
+ *                     type: integer
+ *                   locationId:
+ *                     type: string
+ *                   name:
+ *                     type: string
+ *                   image:
+ *                     type: string
+ *                   street:
+ *                     type: string
+ *                   description:
+ *                     type: string
+ *                   services:
+ *                     type: string
+ *                   updatedAt:
+ *                     type: string
+ *                   createAt:
+ *                     type: string
+ *     responses:
+ *       201:
+ *         description: success
+ */
+router.post(
+  '/hotels',
+  verifyUser,
+  authorize(['travel_administrator', 'super_administrator']),
+  fileService.upload('image'),
+  validation,
+  catchErrors(hotels.createHotel)
+);
+
+/**
+ * @swagger
+ *
+ * /hotels/{hotelId}/rooms:
+ *   post:
+ *     summary: Add Hotel rooms
+ *     description: This allows travel admins to add hotel rooms
+ *     tags:
+ *       - Accommodation
+ *     requestBody:
+ *       content:
+ *         multipart/form-data:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               name:
+ *                 type: integer
+ *               image:
+ *                 type: string
+ *                 format: binary
+ *               description:
+ *                 type: string
+ *               type:
+ *                 type: string
+ *               cost:
+ *                 type: string
+ *     produces:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               status:
+ *                 type: string
+ *               message:
+ *                 type: string
+ *               data:
+ *                 type: object
+ *                 properties:
+ *                   id:
+ *                     type: integer
+ *                   hotelId:
+ *                     type: string
+ *                   name:
+ *                     type: string
+ *                   image:
+ *                     type: string
+ *                   type:
+ *                     type: string
+ *                   description:
+ *                     type: string
+ *                   cost:
+ *                     type: integer
+ *                   status:
+ *                     type: string
+ *                   updatedAt:
+ *                     type: string
+ *                   createAt:
+ *                     type: string
+ *     responses:
+ *       201:
+ *         description: success
+ */
+router.post(
+  '/hotels/:hotelId/rooms',
+  verifyUser,
+  fileService.upload('image'),
+  authorize(['travel_administrator', 'super_administrator']),
```

Comment:
I think you should move this authorize function above the fileUpload function just as you have done with the hotel's endpoint ('/hotels') at line 77 on this file which is working fine when I tested it.